Modules/vc-module-export/VirtoCommerce.ExportModule.Tests/ExportedTypeMetadataTests.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using Xunit;
+using VirtoCommerce.ExportModule.Core.Model;
+using System.Linq;
+
+namespace VirtoCommerce.ExportModule.Tests
+{
+    public class ExportedTypeMetadataTests
+    {
+        [Fact]
+        public async Task GetFromType()
+        {
+            var metadata = ExportedTypeMetadata.GetFromType<Price>();
+            var props = metadata.PropertiesInfo.Select(x => x.Name);
+
+            Assert.Contains("Currency", props); // Check if own property detected
```

Comment:
I think here we need to check all properties from Price  - i mean overall count and presence of specific names.
This should be done to be sure that all props are named correctly, all that were needed were added and no redundant properties were included.
We can simplify test classes, if there would be too many properties. 
We could remove all comments, methods and leave only properties with different types + some collections/entities to test hierarchy.